### ISSUE TITLE
Collection inputs and textarea have no focus styles

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons#12046]
Using a keyboard to focus the form fields the inputs and text area have no focus styling.

<img width="502" alt="stuart_s_collection_ _add-ons_for_firefox" src="https://user-images.githubusercontent.com/1514/42621296-540bfda0-85b5-11e8-9198-3b102c0606ff.png">

See also https://github.com/mozilla/addons-frontend/issues/5573 in-case something generic can be setup that handles this for all fields.
